test/close.test.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const { test } = require('node:test')
+const { join } = require('node:path')
+const { createReadStream } = require('..')
+
+test('should close stream with a callback', (t, done) => {
+  const pathToFile = join(__dirname, 'files', 'test-file-1.txt')
+  const stream = createReadStream(pathToFile, { autoWatch: false })
+
+  stream.on('data', () => {})
+  stream.on('end', () => {
+    stream.close(done)
```

Comment:
calling `stream.close(done)` should not be necessary, it should auto-close on `'end'` or `'close'`.

Reply:
I added this test to test this line behavior:
https://github.com/platformatic/tail-file-stream/blob/eeb90ca299d8ab682a26414415e0b09e5cf7aa68/index.js#L190